### ISSUE TITLE
Unable to set .env SMTP_SECURED to false

### ISSUE THREAD

**Issue by ivoloshin** [LearningLocker/learninglocker#1117]
When setting `SMTP_SECURED=false`, `dotenv` treats all values as string. 

Then it's used as a bool in `lib/connections/nodemailer.js` like so:

```
const smtpConfig = {
  host: process.env.SMTP_HOST,
  port: process.env.SMTP_PORT,
  secure: process.env.SMTP_SECURED,
  auth: {
    user: process.env.SMTP_USER,
    pass: process.env.SMTP_PASS
  }
};
```

I'm pretty sure it's always `true` in this case... It makes it not possible to set to false.

Also, please add `ignoreTLS` value to smtpConfig parameters so we can use simple email relay servers for internal LRSes.

Thanks!

**Comment by ht2** [comment]
Hi @ivoloshin,

I've patched in this PR and added the TLS flags as well. Should be released as v2.2.2 shortly once tests pass.

**Comment by ht2** [comment]
Fixed in https://github.com/LearningLocker/learninglocker/releases/tag/v2.2.2
